### DISCUSSION TITLE
Alterações

### DISCUSSION THREAD

**Discussion by J-A-B-A** [RamonSilva20/mapos#1684] [Ideas]
Boa noite.
Instalei recentemente o mapos, (embora já tenha conhecido a muito tempo). 
Mas como trabalho na área gráfica nunca consegui usar o programa, mesmo assim gostaria de saber como posso proceder  para alterar a ordem de serviço ou venda de forma que me atenda na gráfica.
Desde já que agradecer a atenção, e aguardo caso aguem possa me dar uma direção.

Abraços
Att
João Alexandre.

**Comment by Pr3d4dor** [comment]
@J-A-B-A O que faz com que o mapos não consiga te atender?

**Reply by J-A-B-A** [reply]
Bom dia amigo, atende não, apenas a parte da OS fica muito a desejar para o ramo gráfico, acredito que as outras funções atendam, embora ainda não tenha testado todas elas.
Hoje eu uso o Bling, até agora foi o mais próximo que consegui, atende em quase tudo, mas tive de fazer muitas adaptações e causa muito problema dentro da produção.
Eu acredito que as mudanção na OS para área gráfica seriam poucas, uma vez que o preenchimento da mesma pode ser feita na base do formulário, acontece que precisaria aumentar o banco de dados para acomodar as variáveis da áre agráfica.
Eu consigo fazer isso no Wordpress, mas gostria muito de usar o MAPOS.
Se tiver alguma idéia que me ajude ficarei muito agradecido.

Desde já agradeço o contato.
Att
João Alexandre

**Reply by willph** [reply]
> Bling

Poderia da mais detalhes, por favor

O porque o Mapos não lhe atende?

Quais campos necessários?

O que falta no Mapos?

....

Quanto mais dados e detalhes nos passar, ficara mais fácil de entender.

Obrigado!
Atenciosamente, Equipe Mapos.

**Reply by J-A-B-A** [reply]
Bom dia
Eu trabalho no ramo gráfico a mais de 20 anos, e sempre procurei um programa que me atendesse na parte de venda num todo, nas partes financeiras, controle de contas a pagar e receber principalmente, gração de orçamentos e principalmente na parte de ordem de serviço, o ramo gráfico é um industria e não comércio, mas quando falamos em ordem de serviço o que sempre encontro é algo mais usual, com objetos, então sempre tem o item "DEFEITO".
Na gráfica temos uma vairição de serviços muito grande.
Hoje eu trabalho com o BLING, nas partes de contas a pagar e receber, orçamentos, cobranças e inegrações e le atende bem, muito bem, mas a ordem de serviço, que é praticamente o que eu mais uso no sistema, eu sempre tenho que me adaptar, não existe felixibilidade nos campos e sempre que eu mudo de sistema para melhor atender a produção, levo meses de treinamento e MUUUUUITOS erros.
Sendo assim de tenato bater cabeça resolvi tentar mudar um sistema que já esta quase pronto onde eu tenha pessoas dispóstas a conversar e ajudar.
Descupem pelo texto longo, mas eu acho que se eunão explicar direito fica mais dificil de entender.

Todas as gráficas planas (gráfica plana trabalha com pastas , emveloes, cartão de visita, receituários usam sempre papel, não confunda gráfica com bureau de impressão que vende banner e afesivos em grandes fomatos, elas se autodenominam gráficas mas não são).
Em fim, todas as gráfica planas trabalham com os mesmos requisitos onde entram as variantes do serviço: 

Tipo do serviço: Bloco
                          Talão
                          Pasta
                          etc....

                Papel:Couche-Brilho ou fosco
                         Off-set         
                         SB
                         Autocopiativo
                         etc....

Gramatura do papel: 63g
                                  70g
                                  75g
                                  etc....

Tamanho do trbalho: Váriados

Tamanho do corte: as folhas usdas na grafica na maioria dos trabalhos são de 96x66 cm o tipo ou tamanho do corte vai depender do tamanho do trabalho.

Tipo de acabamento: Talão
                                   Bloco
                                   Picote
                                   Serrilha
                                   Etc....

E quantidade e outras

Para todos os itens acima poide ser feito com formulários pois tudo pode ser previsto.
Hoje para trablhar com bling eu tive de cadastras mais de 1000 ISSO MESMO UM MIL tipos de serviços e ainda assimé necessário fazer ajustes.
Mas como muitos dos clientes da gráfica são de trabalhos recorrentes a clonagem de os facilita muito onde um posso copiar uma os antiga (com novo numero e claro) com as mesmas caracteristicas da anterior.

Seria muito mais simples fazer um formulário de multiplas escolhas dos itens acima que eu mencionei (faltaram alguns que não achei necessário escrever agora).

Bem, espero que isto ajude a entender o que esta faltando.

Desde já agradeço a paciência e atenção

Att
João Alexandre

**Comment by reginalopes** [comment]
Adoro o MapOS, já o uso há algum tempo para minha MINI GRÁFICA RÁPIDA. Porém, eu queria adicionar um check-box, para assinalar se é uma DESPESA OU RECEITA RECORRENTE. Espero que na próxima vcs consigam adicionar essa atualização... Eu tentei, mas não consegui.

Algo assim, tanto para despesa quanto para receita

<img width="602" height="517" alt="image" src="https://github.com/user-attachments/assets/f30514b9-98f8-48a0-a18e-c689134c115b" />

Por outro lado, adicionei uma calculadora no backend, que tem me ajudado muito...
![Screenshot_53](https://github.com/user-attachments/assets/382c8858-5cee-4496-80ce-bc6090016f9d)

**Comment by alexmarolli** [comment]
Poderiam fazer faturamento agrupado, pois tenho vários clientes que fazem fechamento mensal agrupar ordens e vendas para.um fechamento seria ótimo

**Comment by J-A-B-A** [comment]
Bom dia Gianluca

Eu trabalho com gráfica tradicional e gráfica rápida, mais com a
tradicional, para meu trabalho é necessário uma série de configurações e
cálculos para fazer um orçamento, ficaria feliz em ajudar com estes
cálculos e deixar o mapos 100% para gráfica tradicional e comunicação
visual.
Eu poderia descrever no corpo do  pedido da OS todo o serviço, mas lá não
tem local para calcular nada, apenas o valor do produto e seu total.
Para atender esta demanda  MAPOS necessita de vária campos para adicionar
valores de chapa, papel e corte
do formato, quantidade de impressão e muito mais.

Att
João Alexandre

Em sex., 4 de mar. de 2022 às 01:15, Gianluca Bine ***@***.***>
escreveu:

**Comment by gitirana857-pixel** [comment]
Tudo bem amigos Gostaria de saber se e possível mudar a cifra tipo de R$ para $

**Reply by marques823** [reply]
É possível mas não tem configuração ainda, acredito que isso deve ser feito alterando o código.
